W-20905291-rtf-add-video-vidyard-protect-kt (#1622)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/protect-app-properties.adoc

Protecting API Gateway client ID and secret ID is currently supported in Runtime Fabric, either via API or the Anypoint Runtime Manager UI.
--

> Watch this video to see how to protect property values in Runtime Manager through Runtime Manager:

Reviewer: ```suggestion
Watch this video to see how to protect property values by using Runtime Manager:
```

Reviewer: This sentence is confusing with "in Runtime Manager through Runtime Manager". Would the suggestion I gave work, or perhaps "to see how to protect Runtime Manager property values:" ?
